portfolio/src/main/webapp/index.html

```diff
@@ -10,24 +10,63 @@
+            I would always want to dance around and stomp my feet to the rhythms from the perscussion section. Music was and is something that I can feel, the emotion and energy of
+            a piece never fails to move me. When I was eight, I stopped only appreciating music and started to laern how to make some of my own. I started off learning guitar and then
+            over the years I picked up drums and percussion, vocals, bass guitar, and eventually double bass too. I've been a part of various groups like Jazz Band, Chamber Strings,
+            Concert Band, Concert Choir, and an acapella group called Small Choir. Along with the in-school ensembles I've also played in the orchestra pit for 5 different
+            productions. It became a running theme that if I wasn't on the stage acting, I was probably in the pit. Now that I'm in college it has become slightly harder to be as 
+            involved in all the artistic things that I love but I am still in the Jazz Band and have even been playing in my own band called <strong>IMAS</strong>. We placed second 
+            in a Battle of the Jams competition and would have been able to open for a big concert later in the semester had the COVID-19 pandemic not forced all of the universities 
+            to switch to remote learning. 
+            </p>
+        </div>
+      <img src="/images/musicRoom.jpg" alt="Music Room" width="1200" height="351" class="center">
+      <h3 class="center">My Music Room</h3>
+      <img src="/images/bassRehearsal.jpg" alt="Jazz Band Rehearsal" width="800" height="308" class="center">
+      <h3 class="center">Jazz Band Rehearsal</h3>
+      <img src="/images/bassConcert.jpg" alt="Band Concert" width="500" height="333" class="center">
+      <h3 class="center">Band Concert</h3>
+      <img src="/images/IMAS.jpg" alt="IMAS" width="300" height="400" class="center">
+      <h3 class="center">IMAS</h3>
+      <div class="smallContent">
+        <h2>My Pets</h2>
+        <p>I love my pets and I hope you enjoy their portraits below:</p>
+        <img src="/images/Midnight.jpg" alt="Midnight" width="300" height="400" class="center">
+        <h3 class="center">Midnight</h3>
+        <img src="/images/Murray.jpg" alt="Murray" width="300" height="617" class="center">
+        <h3 class="center">Murray</h3>
+        <img src="/images/Fern.jpeg" alt="Fern" width="300" height="402" class="center">
+        <h3 class="center">Fern</h3>
+        <img src="/images/Pumpkin.jpeg" alt="Pumpkin" width="300" height="620" class="center">
+        <h3 class="center">Pumpkin</h3>
+        <img src="/images/Sugar.jpeg" alt="Sugar" width="400" height="301" class="center">
+        <h3 class="center">Sugar</h3>
+        <img src="/images/Norman.jpeg" alt="Norman" width="400" height="225" class="center">
+        <h3 class="center">Norman</h3>
+        </div>
```

Comment:
This </div> should be moved left (un-indented) two spaces to match the opening <div>.